eclipse-collections/src/main/java/org/eclipse/collections/impl/stack/immutable/ImmutableArrayStack.java

```diff
@@ -299,12 +297,6 @@ public boolean containsAllArguments(Object... elements)
         return this.delegate.asReversed().containsAllArguments(elements);
     }
 
-    @Override
-    public MutableStack<T> toStack()
```

Comment:
Do you recall why we did this? Is this safe to replace with the default method?

Reply:
The default implementation lives in StackIterable and is:
```java
    @Override
    default MutableStack<T> toStack()
    {
        return Stacks.mutable.withAllReversed(this);
    }
```

Which is implemented in `MutableStackFactoryImpl`
```java
    @Override
    public <T> MutableStack<T> withAllReversed(Iterable<? extends T> items)
    {
        return ArrayStack.newStackFromTopToBottom(items);
    }
```

So yes it's safe. I don't remember why we wrote this. Sometimes we've created patterns like this as an optimization, but I rarely want to do that. It would have to be for one of the most common iteration patterns and one of the most common collections to make it worth even performance testing.

Reply:
Great, didn't release it was overriding a default method on `StackIterable`. I'm good with this.